config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/builder/ListenPortEntityBuilder.java

```diff
@@ -85,7 +85,7 @@
             "TLS_RSA_WITH_AES_128_GCM_SHA256",
             "TLS_RSA_WITH_AES_128_CBC_SHA256",
             "TLS_RSA_WITH_AES_128_CBC_SHA"));
-    static final List<String> TLS_VERSIONS = asList(TLSV1, TLSV11, TLSV12);
+    public static final List<String> TLS_VERSIONS = asList(TLSV1, TLSV11, TLSV12);
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this member "protected". ([squid:S2386](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS2386))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=98&issues=AWajX2c-rChX7_AlPFEP&open=AWajX2c-rChX7_AlPFEP)

[ISSUE_KEY]: AWajX2c-rChX7_AlPFEP